src/Model/Anon.py

```diff
@@ -819,12 +677,14 @@ def anonymize(path, Datasets, FilePaths, rawdvh):
     DICOM files,
     DHV CSV file,
     Clinical Data CSV file,
+    Pyradiomics CSV file
     and place it in a subdirectory of the specified path
 
     Parameters
     ----------
     path: ``str``
-        Directory in which to place a subdirectory containing the anonymised data
+        The current patient Directory.
```

Comment:
note the change in understanding of which path was actually getting passed.
TODO:  rename argument from path to current_patient_directory